buildSrc/src/main/groovy/com/carrotsearch/gradle/junit4/TestProgressLogger.groovy

```diff
@@ -74,17 +77,14 @@ class TestProgressLogger implements AggregatedEventListener {
     /** Have we finished a whole suite yet? */
     volatile boolean suiteFinished = false
     /* Note that we probably overuse volatile here but it isn't hurting us and
-      lets us move things around without worying about breaking things. */
-
-    TestProgressLogger(Map args) {
-        progressLogger = new ProgressLogger(args.factory.newOperation(TestProgressLogger))
-        progressLogger.setDescription('Randomized test runner')
-    }
+      lets us move things around without worrying about breaking things. */
```

Comment:
Nit: I thought you were OCD too, how does the indentation being off here by one space not drive you crazy?